.buildkite/pipelines/main/launch_unsigned_builders.yml

```diff
@@ -23,7 +23,18 @@ steps:
       buildkite-agent pipeline upload .buildkite/pipelines/main/misc/embedding.yml
       buildkite-agent pipeline upload .buildkite/pipelines/main/misc/llvmpasses.yml
 
-      # Finally, we launch the platform builders (`package_*`) and (`tester_*`) in alphabetical order.
-      buildkite-agent pipeline upload .buildkite/pipelines/main/platforms/linux64.yml
+      # Now, we launch the `linux64` platform builders.
+      rm -rf .buildkite/pipelines/main/platforms/linux_rr.yml
+      cat .buildkite/pipelines/main/platforms/linux.yml >> .buildkite/pipelines/main/platforms/linux_rr.yml
+      cat .buildkite/pipelines/main/platforms/rr.yml    >> .buildkite/pipelines/main/platforms/linux_rr.yml
```

Comment:
Why do you have these split up like this?

Reply:
Okay reading through, I think it's because you want to conditionally use `rr` on certain architectures?  I think the way to do that is to have `linux.yml` always contain the `rr` step, but then use a YAML `if` block to exclude it on certain architectures.

Reply:
> I think it's because you want to conditionally use rr on certain architectures

That's correct.

> I think the way to do that is to have linux.yml always contain the rr step, but then use a YAML if block to exclude it on certain architectures.

I agree - that would be a lot cleaner. I tried to make it work, e.g. something like this:
```yaml
if: "$ARCH" == "63"
```

But Buildkite kept complaining that I had a syntax error in my YAML. The syntax error would go away if I commented out that `if` line.

If you can figure out how to get the `if` working, that would be great! That would be the preferred solution.

Reply:
I couldn't figure out how to get this to work conditionally in Buildkite, so I went back to the old approach of doing the conditional in Julia.

So there is no more separate YAML for `rr` versus non-`rr`. Instead, the `rr_capture.jl` script detects the builder, and uses `rr` if and only id the builder is `tester_linux64`.

Reply:
@staticfloat I think this can be resolved now?

Reply:
Marking this as resolved.